alchemist-web-renderer/src/jvmMain/kotlin/it/unibo/alchemist/server/modules/UtilityModule.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2010-2023, Danilo Pianini and contributors
+ * listed, for each module, in the respective subproject's build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.server.modules
+
+import io.ktor.server.application.Application
+import java.net.InetAddress
+
+/**
+ * Start the default browser of the user on the server address.
+ */
+fun Application.startBrowserModule() {
+    Runtime.getRuntime().exec("xdg-open ${InetAddress.getLocalHost().hostAddress}:$port")
```

Comment:
How can this work on windows / OSX?

Reply:
Plan:
1. I'm merging this one already
2. You might try [this API](https://stackoverflow.com/a/5226244/1916413), that looks more robust
3. Open another PR for the separate update